### PR TITLE
Add WA OTP template example

### PR DESCRIPTION
**Pull request by bld010**
[DEVED-10492](https://twilio-productivity.atlassian.net/browse/DEVED-10492)

This adds a sample from the [docs repo](https://github.com/twilio-internal/docs/tree/4a43d9ca98823e77d1c5942480c686ad1c247e8c/docs/content). 

I may have changed friendly name values, updated for style guide, and I updated the naming of the files to match the requirements of this repo. 

To review: 
- Make sure that the three/four files match logically. The request params match the output params, etc. 
Thanks!

### CODE REVIEW

File: messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "whatsapp/authentication": {
+            "actions": [
+                {
+                    "copy_code_text": "Copy verification code",
+                    "type": "COPY_CODE"
+                }
+            ],
+            "add_security_recommendation": true,
+            "body": "{{1}}",
+            "code_expiration_minutes": 40
```
Comment (micaswyers) [inline messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json:22]: This expires in 40, but in the curl, it's 30?

File: messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "types": {
+        "whatsapp/authentication": {
+            "actions": [
+                {
+                    "copy_code_text": "Copy verification code",
+                    "type": "COPY_CODE"
+                }
+            ],
+            "add_security_recommendation": true,
+            "body": "{{1}}",
```
Comment (micaswyers) [inline messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json:21]: I think I'm missing this -- what is the value that this {{1}} is placeholding for?

Comment (bld010) [reply messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json:21]: Great question ... I had to go look it up myself: https://www.twilio.com/docs/content/whatsappauthentication#send-wa-authentication-templates-created-with-content-templates 

This type of template just has a pre-defined variable, the value of which is passed when sending the API request to send a message with that specific template.

File: messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json (hunk 1 above)
Comment (bld010) [reply messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json:22]: Good catch!

Comment (bld010) [inline messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json:22]:
```suggestion
            "code_expiration_minutes": 30
```

File: messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json (hunk 2 above)
Comment (micaswyers) [reply messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json:21]: Oh sorry, I wasn't seeing that variable defined in the curl sample and got confused. All good!

**Review by micaswyers — APPROVED** [review]

**Pull request merged**